net-misc/hylafaxplus/hylafaxplus-7.0.2.ebuild

```diff
@@ -0,0 +1,165 @@
+KEYWORDS="~amd64 ~x86"
+IUSE="jbig html lcms ldap mgetty pam"
+
+DEPEND="
+	>=sys-libs/zlib-1.1.4
+	app-text/ghostscript-gpl
+	virtual/mta
+	media-libs/tiff:0[jbig?]
+	virtual/jpeg:0
+	jbig? ( media-libs/jbigkit )
+	virtual/awk
+	lcms? ( media-libs/lcms )
+	ldap? (  net-nds/openldap )
+	pam? ( sys-libs/pam )
+	mgetty? ( net-dialup/mgetty[-fax] )
+	!net-dialup/mgetty[fax]
+"
+RDEPEND="${DEPEND}
+	net-mail/metamail
+	!net-dialup/sendpage
+"
+
+S="${WORKDIR}/${MY_P}"
+
+export CONFIG_PROTECT="${CONFIG_PROTECT} /var/spool/fax/etc /usr/lib/fax"
+export CONFIG_PROTECT_MASK="${CONFIG_PROTECT_MASK} /var/spool/fax/etc/xferfaxlog"
+
+PATCHES=(
+	"${FILESDIR}/ldconfig-patch"
+	"${FILESDIR}/${PN}-CVE-2020-1539x.patch"
+	)
+
+src_prepare() {
+	default
+
+	# force it not to strip binaries
+	for dir in etc util faxalter faxcover faxd faxmail faxrm faxstat \
+		hfaxd sendfax sendpage ; do
+			sed -i -e "s:-idb:-idb \"nostrip\" -idb:g" \
+				"${dir}"/Makefile.in || die "sed failed"
+	done
+
+	sed -i -e "s:hostname:hostname -f:g" util/{faxrcvd,pollrcvd}.sh.in || die "sed on hostname failed"
+
+	# Respect LDFLAGS(at least partially)
+	sed -i -e "/^LDFLAGS/s/LDOPTS}/LDOPTS} ${LDFLAGS}/" defs.in || die "sed on defs.in failed"
+
+	sed -i -e "s|-fpic|-fPIC|g" \
+		configure || die
+}
+
+src_configure() {
+	do_configure() {
+		echo ./configure --nointeractive ${1}
+		# eval required for quoting in ${my_conf} to work properly, better way?
+		eval ./configure --nointeractive ${1} || die "./configure failed"
+	}
+	local my_conf="
+		--with-DIR_BIN=/usr/bin
+		--with-DIR_SBIN=/usr/sbin
+		--with-DIR_LIB=/usr/$(get_libdir)
+		--with-DIR_LIBEXEC=/usr/sbin
+		--with-DIR_LIBDATA=/usr/$(get_libdir)/fax
+		--with-DIR_LOCALE=/usr/share/locale
+		--with-DIR_LOCKS=/var/lock
+		--with-DIR_MAN=/usr/share/man
+		--with-DIR_SPOOL=/var/spool/fax
+		--with-DIR_HTML=/usr/share/doc/${P}/html
+		--with-DIR_CGI="${WORKDIR}"
+		--with-PATH_DPSRIP=/var/spool/fax/bin/ps2fax
+		--with-PATH_IMPRIP=\"\"
+		--with-SYSVINIT=no
+		--with-REGEX=yes
+		--with-LIBTIFF=\"-ltiff -ljpeg -lz\"
+		--with-OPTIMIZER=\"${CFLAGS}\"
+		--with-DSO=auto
+		--with-HTML=$(usex html)"
+
+	if use mgetty; then
+		my_conf="${my_conf} \
+			--with-PATH_GETTY=/sbin/mgetty \
+			--with-PATH_EGETTY=/sbin/mgetty \
+			--with-PATH_VGETTY=/usr/sbin/vgetty"
+	else
+		# GETTY defaults to /sbin/agetty
+		my_conf="${my_conf} \
+			--with-PATH_EGETTY=/bin/false \
+			--with-PATH_VGETTY=/bin/false"
+	fi
+
+	#--enable-pam isn't valid
+	use pam || my_conf="${my_conf} $(use_enable pam)"
+	use lcms || my_conf="${my_conf} $(use_enable lcms)"
+	use ldap || my_conf="${my_conf} $(use_enable ldap)"
+	use jbig || my_conf="${my_conf} $(use_enable jbig)"
+
+	tc-export CC CXX AR RANLIB
+
+	do_configure "${my_conf}"
+}
+
+src_compile() {
+	# Parallel building is borked, bug #????
+	emake -j1
+}
+
+src_install() {
+	dodir /usr/{bin,sbin} /usr/$(get_libdir)/fax /usr/share/man
+	dodir /var/spool /var/spool/fax
+	fowners uucp:uucp /var/spool/fax
```

Comment:
You use `fowners` here but don't `DEPEND+RDEPEND` on `acct-*/uucp`.

Reply:
https://gitweb.gentoo.org/data/api.git/tree/files/uid-gid.txt#n73

The `uucp` user and group appear to be provided by baselayout, and can be assumed to exist, right?